db/migrate/20250706201302_add_import_locked_to_feeds.rb

```diff
@@ -0,0 +1,6 @@
+class AddImportLockedToFeeds < ActiveRecord::Migration[7.2]
+  def change
+    add_column :feeds, :import_locked, :boolean, default: true, null: false
+    execute "UPDATE feeds SET import_locked = true"
```

Comment:
It's kind of confusing to me that `podcast.locked` is named so similarly.  Maybe we'll change that to `write_locked` or something someday.  Ah, naming-tech-debt!

Reply:
Agreed - out of scope for this one.  Opened #1290.